src/EditorFeatures/Core/Implementation/Diagnostics/AbstractDiagnosticsTaggerProvider.AggregatingTagger.cs

```diff
@@ -198,6 +210,52 @@ private void OnDiagnosticsUpdated(DiagnosticsUpdatedArgs e)
                 }
             }
 
+            private void ProcessDocumentOrProjectRemoval(DiagnosticsUpdatedArgs e)
+            {
+                if (e.Solution != null)
```

Comment:
+1 on that suggestion as well.  I'll see about making that as part of this change.